snapper/Btrfs.cc

```diff
@@ -1446,19 +1453,19 @@ namespace snapper
     {
 	try
 	{
+	    SDir general_dir = openGeneralDir();
+
 	    if (num == 0)
 	    {
 		SDir subvolume_dir = openSubvolumeDir();
 		subvolid_t id = get_id(subvolume_dir.fd());
-		set_default_id(subvolume_dir.fd(), id);
+		set_default_id(general_dir.fd(), id);
```

Comment:
In C++ there is no single coding style so sometimes they are unfortunately mixed.